src/Compilers/CSharp/Portable/Symbols/ParameterSymbol.cs

```diff
@@ -421,6 +416,8 @@ public virtual bool IsThis
         /// </summary>
         internal abstract DeclarationScope EffectiveScope { get; }
```

Comment:
>EffectiveScope [](http://example.com/codeflow?start=43&length=14)

Should consider removing `EffectiveScope` and using `DeclaredScope` and `HasUnscopedRefAttribute` instead.

Reply:
I didn't understand that. I thought the other difference between effective and declared scope is that we have some implicitly-scoped parameters (`out` in newer LangVer), no?

Reply:
PEParameterSymbol reads this value directly from the packed flags. It seems reasonable to me to keep this as an abstract method on parameter symbols.